### PR TITLE
UI fixes for /account/billing

### PR DESCRIPTION
**Pull request by tiffwong**
## Description

Fixes the placement of the "Edit" button on "Billing Contact" and the "Add Payment Method" button on "Payment Methods" and a regression for the add GooglePay banner.

Screen Shot 2021-08-05 at 6 49 53 PM

## How to test

Go to `/account/billing` and checking the "Billing Contact" and "Payment Method" panels.

### CODE REVIEW

**Review by bnussman — COMMENTED** [review]
Seems to do the job, but I find `minHeight: 0` to be a little weird from a readability standpoint. Maybe `minHeight: 'unset'` could be an alternative or a change could be made to the Button component in the future?

File: packages/manager/src/features/Billing/BillingPanels/PaymentInfoPanel/PaymentInformation.tsx [hunk 1]
```diff
@@ -40,11 +40,16 @@ const useStyles = makeStyles((theme: Theme) => ({
     marginBottom: theme.spacing(3),
   },
   googlePayNoticeContainer: {
+    fontSize: '0.875rem',
```
Comment (bnussman) [inline packages/manager/src/features/Billing/BillingPanels/PaymentInfoPanel/PaymentInformation.tsx:43]: Can you explain the reason for specifying the font size here? Seems like the `'& p'` fontSize is the one being applied to anything

**Review by DevDW — APPROVED** [review]
Looks like the regressions have been fixed 👌 

Although it is like this in prod, should we just have these line up with each other vertically?

Screen Shot 2021-08-05 at 3 59 43 PM

**Review by HanaXu — APPROVED** [review]
I am also seeing the `Edit` button not exactly lining up with the `Billing Contact` paper. Everything else looks fine to me though

image

File: packages/manager/src/features/Billing/BillingPanels/PaymentInfoPanel/PaymentInformation.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (tiffwong) [reply packages/manager/src/features/Billing/BillingPanels/PaymentInfoPanel/PaymentInformation.tsx:46]: @DevDW @bnussman it's to overwrite the font-size that's being passed down from the banner

**Review by bnussman — APPROVED** [review]

**Pull request merged**